src/runtime/storageNG/drivers/volatile.ts

```diff
@@ -45,6 +45,7 @@ export class VolatileStorageKey extends StorageKey {
 
 export class VolatileMemory {
   entries = new Map<string, VolatileEntry<unknown>>();
+  token = Math.random() + '';
```

Comment:
How come the token is a random number here? This isn't immediately obvious to me. (Answering in the form of a comment would be ideal :wink: )